kafka/producer/base.py

```diff
@@ -27,6 +27,9 @@
 
 STOP_ASYNC_PRODUCER = -1
 
+# unlimited
+ASYNC_QUEUE_MAXSIZE = 0
```

Comment:
do you want to have it as _unlimited_ in this case? i thought the whole idea was to have a limit by default.. so maybe you should set it to 65536 in that case...